src/app/core/table/table.component.html

```
@@ -13,7 +13,7 @@
                     <ng-container [matColumnDef]="'actions'" >
                         <th mat-header-cell *matHeaderCellDef  mat-sort-header > {{ `table.col.actions` | translate}} </th>
                         <td mat-cell *matCellDef="let element"> 
```

HTMLHint (1.5.0)

[error] The attribute name of [ *matCellDef ] must be in lowercase.

(attr-lowercase)